### DISCUSSION TITLE
Metaphor Story - A Central Sidebar Menu Configuration | Metaphore (SCP)

### DISCUSSION THREAD

**Discussion by giscus** [StreetCommunityProgrammer/metaphore#86] [Metaphor - Discussion]
# Metaphor Story - A Central Sidebar Menu Configuration | Metaphore (SCP)

darkterminal telling a story about A Central Sidebar Menu Configuration

https://metaphore.vercel.app/stories/javascript/a-central-sidebar-menu-configuration

**Comment by pipifix** [comment]
Hi darkterminal. Thanks for this tutorial. But i got a TS-Error and the menuitems don't show up. i used your code as it comes. the sidebar.tsx is imported in my layout.tsx. TS: gives me this error. "the property "menus" is missing in Type "{}", but is neccessary in Typ "SidebarProps" .ts(2741) sidebar.ts(17, 3): "menus" wird hier deklariert.". Whats wrong with that? the 'menus' in Sidebar.tsx (for map) is empty. When will the sideMenu.tsx (the links) ever read/imported in your code? I'll highly appriciate you help. Thanks, Thomas.

**Comment by darkterminal** [comment]
Thanks for comment, Thomas! Here is you can see the updated version [A Central Sidebar Menu Configuration](https://github.com/darkterminal/a-central-sidebar-menu-configuration). hope this help! :smile:
